### PR TITLE
[ui][fix] Stabilize smoke tests

### PR DESCRIPTION
**Pull request by mcada**
//test: @create-delete-api-connector

### CODE REVIEW

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Comment by mcada** [comment]
retest this please

File: ui-tests/src/test/resources/features/connections/connections.feature [hunk 1]
```diff
@@ -68,7 +68,7 @@ Feature: Connection - CRUD
     Then check visibility of page "Connections"
 
   # now we know there is at least one connection
-    And click on the kebab menu icon of each available connection
+  #  And click on the kebab menu icon of each available connection
```
Comment (avano) [inline ui-tests/src/test/resources/features/connections/connections.feature]: delete if not used

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by avano** [comment]
retest this please

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Pull request merged**